brigadier/accounts/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Forgot tests?
cover lines 17-25 in brigadier/brigadier/accounts/views.py
https://github.com/avoevodin/brigadier/blob/69309dfc11e55c3a14ffa353ea7d2101510c0f2f/brigadier/accounts/views.py#L17-L25